### PR TITLE
Gemfile cleanup

### PR DESCRIPTION
**Pull request by markrickert**

### CODE REVIEW

File: files/Gemfile [hunk 1]
```diff
@@ -10,10 +10,14 @@ gem "afmotion" # Networking
...
-#gem "motion-hockeyapp" 
+#gem "motion-hockeyapp"
 
-# Useful tools 
-#gem "bubble-wrap" 
+# Useful tools
+#gem "bubble-wrap"
+
+# Debugging tools
+#gem "motion_print" # Useful REPL output for objects
```
Comment (squidpunch) [inline files/Gemfile]: `motion_print` is listed as a runtime_dependency, so we don't need that here.

Comment (markrickert) [inline files/Gemfile]: Ahh... didn't realize that :)

It feels weird not having it in my Gemfile though!

**Comment by GantMan** [comment]
Yeah, mp, rmq, and redpotion come out the box :)  Your gem leveled up!

**Comment by GantMan** [comment]
this looks good.

**Comment by GantMan** [comment]
@markrickert if you want, you can metion that motionprint is already included... in a comment.   Perhaps that's a good middleground?

**Comment by markrickert** [comment]
meh.

**Comment by squidpunch** [comment]
maybe a mention in RP's readme!

**Pull request merged**